var/spack/repos/builtin/packages/py-diameter-synthesis/package.py

```diff
@@ -21,7 +22,7 @@ class PyDiameterSynthesis(PythonPackage):
     depends_on('py-numpy@1.15.0:', type='run')
     depends_on('py-scipy@0.13:', type='run')
     depends_on('py-h5py@2.9:', type='run')
-    depends_on('py-morphio@2.7.1:', type='run')
```

Comment:
The `py-morphio` dependency should be kept and updated, because the package make direct use of it. From setup.py:
```
morphio>=2.3.4
```

Reply:
this is my code, I just forgot to update this in setup.py, do you think it is a big deal? I can make a new release of this code to upgrade the setup.py